loader/schemas.py

```diff
@@ -0,0 +1,232 @@
+
+# These schemas are based on the DOS schemas with minor modifications. See
+# https://github.com/ga4gh/data-object-service-schemas/blob/master/openapi/data_object_service.swagger.yaml#L594
+
+data_object_schema = {
```

Comment:
I think this schema should provide an accurate and concise description of what the transformers must produce as output for the data to be successfully loaded into the DSS. Some of the fields identified by the DOS schema as optional are actually required for loading into the DSS, and some fields identified by the DOS schema as required are actually optional for loading into the DSS, and in fact, not even used by the standard loader.
@david4096, what are your thoughts on this?

Reply:
The idea is that this schema should only reflect what is required for loading. Anything non-conforming should be changed (as with the 'name' field that you pointed out earlier).